quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/storage/legacy/infinispan/CacheManagerFactory.java

```diff
@@ -171,6 +183,76 @@ private RemoteCacheManager startRemoteCacheManager() {
         return remoteCacheManager;
     }
 
+    private static void createRemoteCaches(org.infinispan.client.hotrod.configuration.ConfigurationBuilder builder) {
+        // fall back for distributed caches if not defined
+        logger.warn("Creating remote cache in external Infinispan server. It should not be used in production!");
+        var baseConfig = defaultRemoteCacheBuilder().build();
+
+        Stream.of(USER_SESSION_CACHE_NAME, CLIENT_SESSION_CACHE_NAME, ACTION_TOKEN_CACHE, WORK_CACHE_NAME)
+                .forEach(name -> builder.remoteCache(name).configuration(baseConfig.toStringConfiguration(name)));
+
+        // https://issues.redhat.com/browse/ISPN-16270
```

Comment:
Shall we remove the Jira link as the issue doesn't change behaviour required here?

Reply:
I plan to change the default to be non-indexed and use the same configuration in all the caches. Does it make sense? I don't expect any user to use this code to create the caches but it is useful for testing and demos :)